File: csharp/dsf/dsf/src/pool/RawClientInfo.cs
namespace com.baidu.dsf.pool
{

    using ServiceInstance = com.baidu.dsf.register.ServiceInstance;
    using TServiceClient = Thrift.TServiceClient;

    /// <summary>
    /// 保存原生的Thrift客户端对象及服务实例信息
    /// @author arksea
    /// </summary>
    public class RawClientInfo
    {

        public TServiceClient client;
        public ServiceInstance instance;
    }

}

File: java/dsframework/src/main/java/dsf/pool/RawClientInfo.java
package dsf.pool;

import dsf.register.ServiceInstance;

/**
 * 保存原生的Thrift客户端对象及服务实例信息
 * @author arksea
 */
public class RawClientInfo<T> {

    public T client;
    public ServiceInstance instance;
}
